spec/unit/application_spec.rb

```diff
@@ -1,11 +1,11 @@
 require 'rails_helper'
 require 'fileutils'
 
-describe ActiveAdmin::Application do
+RSpec.describe ActiveAdmin::Application do
 
   let(:application) do
     ActiveAdmin::Application.new.tap do |app|
-      # Manually override the load paths becuase RSpec messes these up
+      # Manually override the load paths becuase RSpec.messes these up
```

Comment:
Another bad substitution. Maybe you can correct the "becuase" typo since you're here.